tests/test_breath_detection.py

```diff
@@ -0,0 +1,495 @@
+import copy
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from eitprocessing.datahandling.breath import Breath
+from eitprocessing.datahandling.continuousdata import ContinuousData
+from eitprocessing.datahandling.sequence import Sequence
+from eitprocessing.features.breath_detection import BreathDetection
+
+environment = Path(
+    os.environ.get(
+        "EIT_PROCESSING_TEST_DATA",
+        Path(__file__).parent.parent.resolve(),
+    ),
+)
+data_directory = environment / "tests" / "test_data"
+draeger_file1 = data_directory / "Draeger_Test3.bin"
+draeger_file2 = data_directory / "Draeger_Test.bin"
+timpel_file = data_directory / "Timpel_Test.txt"
+
+
+def make_cosine_wave(sample_frequency: float, length: int, frequency: float):
+    """Generate a cosine wave with the given parameters and amplitude 1.
+
+    A cosine wave starts and ends at a value of 1, passing through -1 halfway between the maximum values. This makes it
+    a very predictable signal for breath detection.
+    """
```

Comment:
internal helper function so I underscored it.

```suggestion
def _make_cosine_wave(sample_frequency: float, length: int, frequency: float):
    """Generate a cosine wave with the given parameters and amplitude 1.

    A cosine wave starts and ends at a value of 1, passing through -1 halfway between the maximum values. This makes it
    a very predictable signal for breath detection and will be used as simplified test data by some of the tests below.
    """
```